README.md

```diff
@@ -1 +1,3 @@
-# vaultpki
\ No newline at end of file
+[![CircleCI](https://circleci.com/gh/giantswarm/vaultpki.svg?&style=shield&circle-token=8adb655cf7ed6250c599312c2c613ba119280ad1)](https://circleci.com/gh/giantswarm/vaultpki)
+
+# vaultpki
```

Comment:
A short summary here would be good and maybe how it relates to the other vault libs.

Reply:
Done.